yarn.lock

```diff
@@ -3153,6 +3153,13 @@
   dependencies:
     regenerator-runtime "^0.13.4"
 
+"@babel/runtime@^7.7.6":
+  version "7.18.6"
+  resolved "https://registry.yarnpkg.com/@babel/runtime/-/runtime-7.18.6.tgz#6a1ef59f838debd670421f8c7f2cbb8da9751580"
+  integrity sha512-t9wi7/AW6XtKahAe20Yw0/mMljKq0B1r2fPdvaAdV/KPDZewFXdaaa6K7lxmZBZ8FBNpCiAT6iHPmd6QO9bKfQ==
+  dependencies:
+    regenerator-runtime "^0.13.4"
```

Comment:
not sure about it, didn't mean to at least. I will review

Reply:
it must be from some of their linked babel dependencies https://github.com/remix-run/react-router/blob/main/package.json

Reply:
Must be this one:
```
=> Found "react-virtualized#@babel/runtime@7.14.8"
info This module exists because "cozy-editor-core#react-virtualized" depends on it.
info Disk size without dependencies: "788KB"
info Disk size with unique dependencies: "832KB"
info Disk size with transitive dependencies: "832KB"
info Number of shared dependencies: 1
```